src/test/resources/detailedNrReportSecurityAnalysis.txt

```diff
@@ -5,72 +5,264 @@
          Network has 2 buses and 2 branches
          Network balance: active generation=603.77 MW, active load=600.0 MW, reactive generation=0.0 MVar, reactive load=200.0 MVar
       + Pre-contingency simulation
-         AC load flow complete with NR status 'CONVERGED'
-        + Newton Raphson on Network CC0 SC0 || No outer loops calculations
+         AC load flow completion status: CONVERGED
+        + Newton Raphson on Network CC0 SC0
+           No outer loops have been launched
           + Initial mismatch
-             Initial mismatch on TargetP: 6.0, Bus Id: 'VL2_0', Bus V: 1.0, Bus Phi: 0.0
-             Initial mismatch on TargetQ: 1.382400000000004, Bus Id: 'VL2_0', Bus V: 1.0, Bus Phi: 0.0
-             Initial mismatch on TargetV: 0.0050000000000000044, Bus Id: 'VL1_0', Bus V: 1.0, Bus Phi: 0.0
              Norm |f(x0)|=6.157195364774453
-          + Iteration 0 mismatch
-             Iteration 0 mismatch on TargetP: 0.198586549381778, Bus Id: 'VL2_0', Bus V: 0.9747979766262653, Bus Phi: -0.0605498160569332
-             Iteration 0 mismatch on TargetQ: 0.2032848645560701, Bus Id: 'VL2_0', Bus V: 0.9747979766262653, Bus Phi: -0.0605498160569332
-             Iteration 0 mismatch on TargetV: 0.0, Bus Id: 'VL1_0', Bus V: 0.995, Bus Phi: -3.837262538969321E-18
+            + Mismatch on TargetP : 6.0
```

Comment:
Personally, I also prefer here "mismatch on balanceP" since the mismatch term refers to an equation, and the equation here is the Kirchhoff law.

Reply:
agree, changed to ` Largest P mismatch` (same for Q, V), which i also think is indeed better